duckduckgo/macos-browser
# Fire button

Task/Issue URL: https://app.asana.com/0/72649045549333/1178859670483591/f
Tech Design URL: https://app.asana.com/0/481882893211075/1199096691990535/f
CC:

**Description**:
When a user clicks on the burn button, it triggers a confirmation dialog. After the confirmation all windows and tabs are closed + cookies are removed.

**Steps to test this PR**:
1. Test the burn button with more windows and tabs open
2. Make sure cookies are cleared properly
3. Run unit tests


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
